packages/backend/jest.config.js

```diff
@@ -1,7 +1,29 @@
-module.exports = {
+const common = {
     preset: 'ts-jest',
     testEnvironment: 'node',
     automock: false,
     setupFiles: ['./setupJest.ts'],
-    testPathIgnorePatterns: ['/node_modules/', '/dist/'],
+    moduleNameMapper: {
+        '^@/(.*)$': '<rootDir>/src/$1',
```

Comment:
okay that means we do use it.. but I don't see it anywhere

Reply:
I did a quick search, it doesn't look like we actively use it, but it's also added in the tsconfig. Maybe it was a default project configuration.